environments/prod/pre-recorded-evidence-justice-gov-uk.yml

```
@@ -11,7 +11,10 @@ srv: []
 cname: 
   - name: "portal-sbox"
     ttl: 300
-    record: "pre-sbox.powerappsportals.com"
+    record: "sdshmcts-sbox.azurefd.net"
+  - name: "afdverify.portal-sbox"
+    ttl: 300
+    record: "afdverify.hmcts-sbox.azurefd.net"
```

I think it must be this.
```suggestion
    record: "afdverify.sdshmcts-sbox.azurefd.net"
```